.ci/containers/github-differ/test_tools.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+MM_LOCAL_PATH=$1
+TPG_LOCAL_PATH=$2
+mm_commit_sha=$3
+build_id=$4
+build_step=$5
+project_id=$6
+
+github_username=modular-magician
+
+pushd $MM_LOCAL_PATH/tools/breaking-change-detector
+go test
```

Comment:
```suggestion
go test ./...
```

We'll want to use `./...` to the tests in the subfolders are run